python/rikai/spark/sql/codegen/base.py

```diff
@@ -193,6 +193,10 @@ def udf_from_spec(spec: ModelSpec):
     elif spec.flavor == "sklearn":
         from rikai.spark.sql.codegen.sklearn import generate_udf
 
+        return generate_udf(spec)
+    elif spec.flavor == "sparkml":
```

Comment:
`sparkml` -> `spark`

https://www.mlflow.org/docs/latest/models.html#spark-mllib-spark

It would be better to keep the flavor name as it is in MLflow.